src/plugins/vis_default_editor/public/components/controls/time_interval.tsx

```diff
@@ -110,21 +111,22 @@ function TimeIntervalParamEditor({
   const timeBase: string = get(editorConfig, 'interval.timeBase') as string;
   const options = timeBase
     ? []
-    : ((aggParam as any).options || []).reduce(
-        (filtered: ComboBoxOption[], option: AggParamOption) => {
-          if (option.enabled ? option.enabled(agg) : true) {
-            filtered.push({ label: option.display, key: option.val });
-          }
-          return filtered;
-        },
-        [] as ComboBoxOption[]
-      );
+    : (aggParam.options || []).reduce((filtered: ComboBoxOption[], option: AggParamOption) => {
+        if (option.enabled ? option.enabled(agg) : true) {
+          filtered.push({ label: option.display, key: option.val });
+        }
+        return filtered;
+      }, []);
 
   let selectedOptions: ComboBoxOption[] = [];
   let definedOption: ComboBoxOption | undefined;
   if (value) {
     definedOption = find(options, { key: value });
-    selectedOptions = definedOption ? [definedOption] : [{ label: value, key: 'custom' }];
+    selectedOptions = definedOption
+      ? [definedOption]
```

Comment:
nit: I'm not a big fan of constructions like that `x < y ? x < z ? x : z : y < z ? y : z; ` It's a kind of anti-pattern I think